content/integrations/google_cloud_platform.md

```diff
@@ -11,7 +11,7 @@ Connect to Google Cloud Platform to:
 
 * See your Google Compute Engine hosts in the infrastructure overview
 * Import your Google Compute Engine host tags
-* Tag your Google Compute Engine hosts with additional compute-specific metadata (e.g. zone)
+* Tag your Google Compute Engine hosts with additional GCP labels (only `key:value` label pattern are collected and defined as a tag)
```

Comment:
Zone is already collected as tag (cf end of the doc page) I'm going to find an other example.

We could do a KB/FAQ about it, what do you think ?